src/test/java/org/myrobotlab/test/AbstractTest.java

```diff
@@ -167,31 +163,31 @@ static protected void installAll() {
    */
   public static void releaseServices() {
 
-    log.info("end of test - id {} remaining services {}", Runtime.getInstance().getId(), Arrays.toString(Runtime.getServiceNames()));
+    log.info("end of test - id {} remaining services {}", Runtime.getInstance().getId(),
+        Arrays.toString(Runtime.getServiceNames()));
 
     // release all including runtime - be careful of default runtime.yml
     Runtime.releaseAll(true, true);
     // wait for draining threads
     sleep(100);
-    // resets runtime with fresh new instance
-    Runtime.getInstance();
 
     // check threads - kill stragglers
     // Set<Thread> stragglers = new HashSet<Thread>();
     Set<Thread> threadSetEnd = Thread.getAllStackTraces().keySet();
     Set<String> threadsRemaining = new TreeSet<>();
     for (Thread thread : threadSetEnd) {
-      if (!threadSetStart.contains(thread) && !"runtime_outbox_0".equals(thread.getName()) && !"runtime".equals(thread.getName())) {
-          threadsRemaining.add(thread.getName());
+      if (!threadSetStart.contains(thread) && !"runtime_outbox_0".equals(thread.getName())
+          && !"runtime".equals(thread.getName())) {
+        threadsRemaining.add(thread.getName());
       }
     }
     if (threadsRemaining.size() > 0) {
       log.warn("{} straggling threads remain [{}]", threadsRemaining.size(), String.join(",", threadsRemaining));
     }
 
-    // log.warn("end of test - id {} remaining services after release {}",
-    // Platform.getLocalInstance().getId(),
-    // Arrays.toString(Runtime.getServiceNames()));
+    // resets runtime with fresh new instance
+    Runtime.getInstance();
```

Comment:
I'm surprised at the comment here... getInstance() ,  I would expect a getInstance() call to reset the state of Runtime ...

Reply:
Its fresh, because the teardown of an AbtractTest now tears down Runtime as well .. which it previously didn't ...
We left runtime hanging out previously